### DISCUSSION TITLE
Clarification about what's the technique used under the hood

### DISCUSSION THREAD

**Discussion by AlaDouagi** [stitchesjs/stitches#769] [Ideas]
**Is your feature request related to a problem? Please describe.**
In the documentation there is nothing clear about the exact technique used inside the library's core, is it CSS extraction or simple CSS in JS or a mix of both?

**Describe the solution you'd like**
The technic used in the core needs to have a full page in the docs that talk about it and describes what's under the hood and why not include a comparison with other libraries (techniques, benchmarks...)

**Describe alternatives you've considered**
Digging inside the repo's codebase is an option but an unintuitive one.

**Additional context**
The onboarding pages in the docs need some work, this issue is a good one to start with.

**Comment by peduarte** [comment]
Hi!

> In the documentation there is nothing clear about the exact technique used inside the library's core, is it CSS extraction or simple CSS in JS or a mix of both?

I don't know what you mean. Can you give me examples of other styling libraries that clearly describe this to you, so I can understand better?

> The technic used in the core needs to have a full page in the docs that talk about it and describes what's under the hood

Why? This seems like going too deep into the technical implementation. Does it matter?

> why not include a comparison with other libraries (techniques, benchmarks...)

Have you seen our [benchmark page](https://stitches.dev/docs/benchmarks), and our [blog posts](https://stitches.dev/blog) (there are posts about migration from Emotion/SC to Stitches, and outlines their differences)?

> Digging inside the repo's codebase is an option but an unintuitive one.

This is one of the coolest things about open source. That you can dig into the code, and learn, or help improve. Are you saying the code base is unintuitive?

> The onboarding pages in the docs need some work, this issue is a good one to start with.
What kind of work specifically? Open to constructive feedback.

**Comment by AlaDouagi** [comment]
Hello @peduarte, what I meant is a good thing to have a clear technical overview of the library, developers that want to consume the library would certainly like to know the tradeoffs and what's the runtime cost of it with a technical explanation provided, here is an example of a page that I wanted to be included in this library's docs: https://chakra-ui.com/docs/comparison

**Comment by peduarte** [comment]
@AlaDouagi Gotcha. Thanks for the reference.
